js/functions.js

```diff
@@ -103,7 +103,9 @@
 
 	// Add a class to big image and caption larger than or equal to 840px.
 	function bigImageClass() {
-		if ( $body.hasClass( 'page' ) || $body.hasClass( 'search' ) || $body.hasClass( 'single-attachment' ) || $body.hasClass( 'error404' ) ) {
```

Comment:
Couldn't this be simplified to `if ( $body.is( '.page, .search, .single-attachment, .error404' ) ) {`?